QUICK_START_GRPC.md

```diff
@@ -94,26 +94,50 @@ Let's now add vectors with some payload:
 
 ```bash
 grpcurl -plaintext -import-path ./lib/api/src/grpc/proto/ -proto qdrant.proto -d '{
-  "collection_name": "test_collection",
+  "collection_name": "milittle",
   "wait": true,
   "points": [
     {
       "id": {
         "num": 1
-      },
-      "vector": [0.05, 0.61, 0.76, 0.74],
-      "payload": {
-        "city": "Berlin",
-        "country": "Germany",
-        "population": 1000000,
-        "square": 12.5,
-        "coords": { "lat": 1.0, "lon": 2.0 }
+      }, 
+      "vectors": { 
+        "vector": { 
+          "data": [0.05, 0.61, 0.76, 0.74] 
+        } 
+      }, 
+      "payload": 
+      {
+        "city": { 
+          "string_value": "Berlin" 
+        }, 
+        "country": { 
+          "string_value": "Germany" 
+        }, 
+        "population": { 
+          "integer_value": 1000000 
+        }, 
+        "square": { 
+          "double_value": 12.5 
+        }, 
+        "coords": { 
+          "struct_value": {
+            "fields":{ 
+              "lat": { 
+                "double_value":1.0 
+              }, 
+              "lon": { 
+                "double_value": 2.0 
+              } 
+            }
+          }
+        }
```

Comment:
We should probably compact it like this to prevent the example from exploding (and the same for `id`):

```suggestion
      },
      "vectors": {
        "vector": { "data": [0.05, 0.61, 0.76, 0.74] }
      },
      "payload": {
        "city": { "string_value": "Berlin" },
        "country": { "string_value": "Germany" },
        "population": { "integer_value": 1000000 },
        "square": { "double_value": 12.5 },
        "coords": {
          "struct_value": {
            "fields":{
              "lat": { "double_value": 1.0 },
              "lon": { "double_value": 2.0 }
            }
          }
        }
```

Also, a lot of your changed lines have trailing spaces, this should not happen.

Reply:
Okay, got it.